tests/Mappers/Parameters/TypeMapperTest.php

```diff
@@ -123,6 +140,14 @@ private function dummy()
 
     }
 
+    /**
```

Comment:
Remove unused code

Reply:
This method is used by the test case (`testParameterWithDescription`), like others here